### PR TITLE
Create scheduler.md

### PR DESCRIPTION
**Pull request by jeanniefinks**
new content for single and multi-stream scheduling

### CODE REVIEW

File: docs/source/scheduler.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+_Multi-stream scheduling; requests execute in parallel and may utilize hardware resources better_
+
+Whereas the default scheduler will queue up requests made simultaneously and handle them serially, the multi-stream scheduler maintains a set of dropboxes where requests may be deposited and the requesting threads can wait. These dropboxes allow workers to find work from multiple sources when work from a single source would otherwise be scarce, maximizing throughput. When a request is complete, the requesting thread is awakened and returns the results to the caller.
+
+The most common use cases for the multi-stream scheduler are where parallelism is low with respect to core count, and where requests need to be made asynchronously without time to batch them. Implementing a model server may fit such a scenario and be ideal for using multi-stream scheduling.
+
+Depending on your engine execution strategy, enable one of these options by running:
+
+```
```
Comment (mgoin) [inline docs/source/scheduler.md:43]:
```suggestion
```python
```

File: docs/source/scheduler.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+engine = compile_model(model_path, batch_size, num_cores, num_sockets, "single_stream")
+```
+
+or
+
+```
```
Comment (mgoin) [inline docs/source/scheduler.md:49]:
```suggestion
```python
```

**Review by mgoin — COMMENTED** [review]
LGTM! just suggestions for python formatting in doc

**Review by Willtor — APPROVED** [review]
This looks good, Jeannie. Thanks very much!

**Review by mgoin — APPROVED** [review]
thanks @jeanniefinks !

**Pull request merged**